021-IdentityForApps/Student/05-claims-enrichment.md

```diff
@@ -26,7 +26,8 @@ You will call the Verify-inator at its `/Territory` endpoint. So if you deployed
 
 Lastly, CMC IT Leadership does not want to present the "Territory Name" attribute to the user during sign-up. They have asked you to please remove this field from the sign-up experience.
 
-**NOTE:** This ASPNETCORE web api application is located in the folder named **Verify-inator** in your Teams channel Files tab.
+**NOTE:** This ASPNETCORE web api application is located in the **Verify-inator** folder under the [student resources](Resources/Verify-inator) or in your Teams channel Files tab.
```

Comment:
As per the WTH Author's guide, please remove the link to the folder in the WTH repo.   Instead, refer to the relative path in the Resources.zip file provided by the coach.

Reply:
Fixed!